---
title: Bloco 'Catch' nunca alcançado; <exception> manipulado acima na mesma instrução 'Try'
ms.date: 07/20/2015
f1_keywords:
- bc42031
- vbc42031
helpviewer_keywords:
- BC42031
ms.assetid: 7d15597c-5988-42ea-a853-63cbf78faaf3
ms.openlocfilehash: 0fcb4db60ecc8236293e74409727b20b5fccdceb
ms.sourcegitcommit: 14355b4b2fe5bcf874cac96d0a9e6376b567e4c7
ms.translationtype: MT
ms.contentlocale: pt-BR
ms.lasthandoff: 01/30/2019
ms.locfileid: "55265370"
---
# <a name="catch-block-never-reached-exception-handled-above-in-the-same-try-statement"></a>Bloco 'Catch' nunca alcançado; \<exceção > manipulado acima na mesma instrução 'Try'
Um `Catch` bloco no código não pode ser acessado porque ela é tratada em precedidos `Try` bloco.  
  
 Por padrão, esta mensagem é um aviso. Para obter mais informações sobre como ocultar avisos ou tratar avisos como erros, consulte [Configurando avisos no Visual Basic](/visualstudio/ide/configuring-warnings-in-visual-basic)  
  
 **ID do erro:** BC42031  
  
## <a name="to-correct-this-error"></a>Para corrigir este erro  
  
-   Remova a instrução redundante.  
  
## <a name="see-also"></a>Consulte também
- [Como: Capturar uma exceção no Visual Basic](https://msdn.microsoft.com/library/f3063c89-d2bf-49b1-91b5-b87edfb18b95)
- [Como: Testar o código com um bloco Try... Bloco Catch no Visual Basic](https://msdn.microsoft.com/library/8368e205-ed73-4185-a247-af84fb4fafa9)
- [Como: Erros de filtro em um bloco Catch no Visual Basic](https://msdn.microsoft.com/library/85964d0a-56e7-4301-a96e-5eaea23b7b9b)
- [Passo a passo: (Visual Basic) de tratamento de exceções estruturado](https://msdn.microsoft.com/library/440da655-4b32-490b-8b16-bfe46f41fa76)
- [Instrução Try...Catch...Finally](../../visual-basic/language-reference/statements/try-catch-finally-statement.md)
